test/3rdparty/concurrencytest-0.1.2/concurrencytest.py

```diff
@@ -43,6 +45,23 @@
 
 CPU_COUNT = cpu_count()
 
+def wait_for_children(pids):
+    while pids:
+        try:
+            # Loop through each child pid to support windows (plus it won't be a large number).
```

Comment:
Does this add support for parallel processing on Windows via the concurrency library? We've worked around that for years..
See WT-3656 and WT-2815 (I didn't look deeply, just saw the change in this file and comment about Windows)

Reply:
No it doesn't. Our current solution on windows is to create buckets of unit-tests and run them separately. This works well there for now. My comment here is more for future changes if this is used on Windows (based on the python documentation how Windows requires a PID and we can't use -1 as all children).